apps/simple-trading-app/src/app/components/drawer/drawer.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { theme } from '@vegaprotocol/tailwindcss-config';
+import type { ReactElement } from 'react';
+import { useEffect } from 'react';
+import Drawer from '@mui/material/Drawer';
+
+interface Props {
+  children?: ReactElement | ReactElement[];
+  isMenuOpen?: boolean;
+  onToggle(): void;
+}
+
+export const NavigationDrawer = ({
+  isMenuOpen = false,
+  onToggle,
+  children,
+}: Props) => {
+  const [windowSize, setWindowSize] = React.useState({
+    width: window.innerWidth,
```

Comment:
non-blocking: suggestion for the future - something like this (resolutions of the viewport getter etc.) would be useful to have in external hook or provider

Reply:
👍